dumpling/dumpling.go

```diff
@@ -104,7 +104,11 @@ func (m *Dumpling) Process(ctx context.Context, pr chan pb.ProcessResult) {
 		errs = append(errs, unit.NewProcessError(err))
 	}
 
-	m.logger.Info("dump data finished", zap.Duration("cost time", time.Since(begin)))
+	if err == nil {
```

Comment:
How about checking `errs` as we send it to `pb.ProcessResult`.

Reply:
addressed in https://github.com/pingcap/dm/pull/661/commits/8beb823b1ddc11c32a40345d363bf3168b2a1157